host-controller/src/main/java/org/jboss/as/host/controller/HostModelUtil.java

```diff
@@ -97,6 +97,7 @@ public static void createRootRegistry(final ManagementResourceRegistration root,
 
         // Other root resource operations
         root.registerOperationHandler(CompositeOperationHandler.DEFINITION, CompositeOperationHandler.INSTANCE);
+        //root.registerOperationHandler(MapReduceOperationHandler.DEFINITION, MapReduceOperationHandler.INSTANCE);
```

Comment:
Please remove. The original ordering of the imports was the correct one as well.